### DISCUSSION TITLE
Using multiple page repos on same apex domain with subdomain

### DISCUSSION THREAD

**Discussion by 8bitAlex** [community/community#51617] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I looking to see if there is a way to support two page repos on a single domain. Initially I wanted to have `alexsalerno.com` as my main page and a blog as `blog.alexsalerno.com`. However, I see according to the docs that it is not supported. Presumably because of some issue with SSL certs.

If I ignore that advice and try and set it up anyways I can get both repos to work just fine with the exception that the subdomain doesn't support SSL.

If instead I try and use the following `alexsalerno.com/blog` which is where my blog repo points to by default I get failed to load resources.

Which looks something like this
<img width="1306" alt="Screenshot 2023-03-31 at 3 16 47 PM" src="https://user-images.githubusercontent.com/4384235/229242528-01088a15-16e5-4d33-8525-c346e436e616.png">

Here are my DNS records
<img width="768" alt="Screenshot 2023-03-31 at 3 18 50 PM" src="https://user-images.githubusercontent.com/4384235/229242726-14287acf-b448-46fb-abbc-eac8dc1c2684.png">

This has to be a common use case. I'm actually surprised that there isn't some way to have both apex and subdomain support. Anyways, any help would be appreciated.

**Comment by josecelano** [comment]
Hi @8bitAlex did you solve it? I have the same problem.
